packages/hash/integration/src/tests/integration.test.ts

```diff
@@ -969,33 +973,37 @@ describe("logged in user ", () => {
       });
 
       // There's many entities from the ones added before this test
-      expect(entities.length).toEqual(24);
-      expect(entities.map((ents) => ents.entityTypeName)).toEqual([
-        "Page",
-        "Block",
-        "Divider",
-        "Block",
-        "Text",
-        "Block",
-        "Text",
-        "Page",
-        "Text",
-        "Block",
-        "Block",
-        "Text",
-        "Text",
-        "Block",
-        "Page",
-        "Block",
-        "Unknown",
-        "Block",
-        "Unknown",
-        "Text",
-        "Text",
-        "Block",
-        "Block",
-        "User",
-      ]);
```

Comment:
I disagree with this test, as the order in which a block and its "block entity" are created as part of `updatePageContents` should not matter (ideally these are created concurrently), so if the order changes the tests should not break. Have added tests for checking whether the correct amount of each entity has been created instead.

Reply:
Yeah I agree. I think It's better not to assume the order is deterministic :+1: